bin/deploy

```diff
@@ -10,6 +10,17 @@ if [[ $# -ne 1 ]]; then
 	echo "Please specify an environment."
 	exit 1
 else
-	bundle exec middleman build --clean --environment="$ENV"
-	bundle exec middleman s3_sync --environment="$ENV"
+  if [[ $ENV =~ pro ]]
+  then
+    read -p "You are about to deploy to production. Are you sure? (y/n) " -n 1 -r
+    echo    # move to a new line
+    if [[ $REPLY =~ ^[Yy]$ ]]
+    then
```

Comment:
Let's join if/then lines like we do at the top of the script.
`if [[ $REPLY =~ ^[Yy]$ ]]; then`